joomla/joomla-cms
# Reduce the number of page duplicates in com_content categories when pagination is used.

### Summary of Changes

Reduce the number of page duplicates in com_content categories when pagination is used.
### Testing Instructions

[https://www.joomla.org/announcements.html?start=999999999999999999999](https://www.joomla.org/announcements.html?start=999999999999999999999) - it opens, while it should not. With this patch applied, pages will open if they really exist: [https://www.joomla.org/announcements.html?start=390](https://www.joomla.org/announcements.html?start=390) - this will be the last page.
